examples/v1beta1/windows-custom-userdata.yaml

```diff
@@ -0,0 +1,40 @@
+# This example provisioner will provision instances using the Windows 2022 EKS-Optimized AMI.
```

Comment:
(nit) When a YAML file has a `---` line, I put one at the start as well.

That helps people learn that `---` is a document start marker, not a separator.